experiment/references.md

```diff
@@ -1 +1,19 @@
-### Link your references in here
\ No newline at end of file
+1. Gel Electrophoresis Virtual Lab:[reference-link](http://learn.genetics.utah.edu/content/labs/gel/)  
```

Comment:
Multiple http links used. Please strictly use https. If there's no https working links available, comment out the reference with a comment